### PR TITLE
Update ocpr-workload-ccnrd-cuttingedge to update RHBQ 1.11

### PR DESCRIPTION
**Pull request by danieloh30**
##### SUMMARY
Update ocpr-workload-ccnrd-cuttingedge to update RHBQ 1.11

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: ansible/roles/ocp4-workload-ccnrd-cuttingedge/tasks/remove_workload.yml [hunk 1]
```diff
@@ -43,6 +43,11 @@
   with_list: "{{ users }}"
   ignore_errors: yes
 
+- name: remove the clusterrole che-sa
+  shell:
```
Comment (stencell) [inline ansible/roles/ocp4-workload-ccnrd-cuttingedge/tasks/remove_workload.yml:47]: Can you use the k8s module for this with `absent` ?

**Review by stencell — CHANGES_REQUESTED** [review]
Mostly version changes, but one comment to address please.

File: ansible/roles/ocp4-workload-ccnrd-cuttingedge/tasks/remove_workload.yml (hunk 1 above)
Comment (danieloh30) [reply ansible/roles/ocp4-workload-ccnrd-cuttingedge/tasks/remove_workload.yml:47]: Can we delete clusterrole with `absent`?

Comment (danieloh30) [reply ansible/roles/ocp4-workload-ccnrd-cuttingedge/tasks/remove_workload.yml:47]: Never tried it before.

Comment (danieloh30) [reply ansible/roles/ocp4-workload-ccnrd-cuttingedge/tasks/remove_workload.yml]: Ok. Just tested with `absent` and it works.

Thanks for the reivew.

**Review by stencell — APPROVED** [review]

**Pull request merged**